logging/export.js

```diff
@@ -14,32 +14,38 @@
 'use strict';
 
 // [START setup]
+// You must set these environment variables to run this sample
 var projectId = process.env.TEST_PROJECT_ID;
 var keyFilename = process.env.GOOGLE_APPLICATION_CREDENTIALS;
 
+// If you don't set the environment variables, then you can modify this file
+// to set the values
 projectId = projectId || '<your-project-id>';
 keyFilename = keyFilename || '/path/to/keyfile.json';
 
+// Provide projectId and authentication to gcloud
 var gcloud = require('gcloud')({
   projectId: projectId,
   keyFilename: keyFilename
```

Comment:
keyfilename is optional, as it can automatically use application default credentials.